src/components/header/search/Search.jsx

```diff
@@ -15,10 +15,10 @@ import {
     search,
 } from '../../../actions/search';
 
-
+const field = "top"
```

Comment:
I would call it `FIELD_NAME` to clearly denote everywhere that it's a (global) constant.

Reply:
Fixed by 9a470f15a85febd6ff8f9c776f3a040a8a2ca45c